docs/ref/contrib/postgres/fields.txt

```diff
@@ -517,6 +517,14 @@ excluded, that is ``[)`` (see the PostgreSQL documentation for details about
 fields (:class:`.DateTimeRangeField` and :class:`.DecimalRangeField`) by using
 the ``default_bounds`` argument.
 
+.. warning::
+
+    PostgreSQL normalizes a range with no points to an empty range. This means
+    that a range with equal values specified for an included lower bound and
+    an  excluded upper bound (for example, ``Range(datetime.date(2005, 6, 21),
+    datetime.date(2005, 6, 21))`` or ``[4, 4)``) will become empty after saving
+    to the database, and the original bound values will be lost.
```

Comment:
```suggestion
.. admonition:: PostgreSQL normalizes a range with no points to the empty range

    A range with equal values specified for an included lower bound and an
    excluded upper bound, such as ``Range(datetime.date(2005, 6, 21),
    datetime.date(2005, 6, 21))`` or ``[4, 4)``, has no points. PostgreSQL will
    normalize the value to empty when saving to the database, and the original
    bound values will be lost. See the `PostgreSQL documentation for details
    <https://www.postgresql.org/docs/current/rangetypes.html#RANGETYPES-IO>`_.
```
I think let's have a titled admonition rather than a warning and lets include the documentation link